### ISSUE TITLE
a11y: "Close list box" button in dropdown is not screen reader accessible (Android)

### ISSUE THREAD

**Issue by staceytan1998** [opengovsg/FormSG#5878]
**Issue:** 
Pressing the "Close list box" button on list box questions (e.g. Q7) does not work using a screen reader.

Fortunately, the screen reader focus is not trapped in the list box. The user is still able to move on to other questions and activate other form fields. However, this can be visually very confusing for some users who still partially rely on sight as the "close list box" button will appear not to work.

Note: This issue does not seem to occur on the Android Firefox browser. The issue does occur on the Android Chrome browser.

**Suggestion:**
Ensure the "Close list box" button can be activated with screen readers. It is not immediately clear why exactly this button functionality does not work specifically in the Android Chrome browser.

**Screenshot:**
https://drive.google.com/file/d/1UFzGN5QoesN9EsvkXma2zFSSnOMI45PE/view?usp=share_link

**Comment by staceytan1998** [comment]
### Investigation Notes
- Not replicable on iOS and Mac VO
